### DISCUSSION TITLE
默认关闭B站合集的“自动连播”

### DISCUSSION THREAD

**Discussion by dupontjoy** [the1812/Bilibili-Evolved#3705] [功能建议 / Ideas]
能否做个组件，把B站视频合集的“自动连播”，默认关掉啊
![Snipaste_2022-09-24_18-07-43](https://user-images.githubusercontent.com/4965288/192092310-6d72b509-8f8a-4207-8430-1fbf1cc64a67.png)

**Comment by Geeyun-JY3** [comment]
这玩意不是记忆的么

**Reply by dupontjoy** [reply]
是每个合集单独记忆的
所以想要一个一次性关闭的

**Reply by Geeyun-JY3** [reply]
今天我实测这玩意还是和「播放方式」设置项对应，「播完暂停」对应「自动连播」关闭

**Reply by dupontjoy** [reply]
「播完暂停」在哪里设置的？找了一圈没找到

**Reply by Geeyun-JY3** [reply]
播放器控制栏齿轮 > 更多播放设置
![image](https://user-images.githubusercontent.com/49396421/192436873-11e8afc3-1874-4da8-9b6a-13434a7a990b.png)
![image](https://user-images.githubusercontent.com/49396421/192436764-747aa513-f3f9-44cf-a8c6-b7c692417ef4.png)

**Reply by dupontjoy** [reply]
哦哦，试了下是联动的
多謝

**Reply by Cyiming** [reply]
> 哦哦，试了下是联动的 多謝

没用的 还是连播

**Reply by Cyiming** [reply]
> 这玩意不是记忆的么

不是记忆 只当天有效 第二天又连播
